data_steward/validation/metrics/required_labs.py

```diff
@@ -54,6 +80,67 @@ def load_measurement_concept_sets_descendants_table(project_id, dataset_id):
     :return: None
     """
 
+    descendants_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+
+    client = bq.get_client(project_id)
+    dataset = client.dataset(dataset_id)
+    vocab_dataset = client.dataset(common.VOCABULARY_DATASET)
+
+    # concept table and concept ancestor table source tables
+    concept_source_table = vocab_dataset.table(common.CONCEPT)
+    concept_ancestor_source_table = vocab_dataset.table(common.CONCEPT_ANCESTOR)
+
+    # concept table and concept ancestor table destination tables
+    concept_dest_table = dataset.table(common.CONCEPT)
+    concept_ancestor_dest_table = dataset.table(common.CONCEPT_ANCESTOR)
+
+    descendants_table_ref = dataset.table(
+        MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE)
+    concept_table_ref = dataset.table(common.CONCEPT)
+    concept_ancestor_table_ref = dataset.table(common.CONCEPT_ANCESTOR)
+
+    # query will check the row counts of each table in the specified dataset
+    row_count_query = ROW_COUNT_QUERY.render(project_id=project_id,
+                                             dataset_id=dataset_id)
+
+    job = client.query(row_count_query)
+    results = job.result()
+
+    # will check if either the CONCEPT and CONCEPT_ANCESTOR tables is empty or not
+    # If so, the CONCEPT and CONCEPT_ANCESTOR tables will be copied from the common.VOCABULARY
+    # table to the destination dataset
+    if results.total_rows == 0:
+        client.copy_table(concept_source_table, concept_dest_table)
+        client.copy_table(concept_ancestor_source_table,
+                          concept_ancestor_dest_table)
+    else:
+        pass
```

Comment:
You might want to use/create the `results` as a dataframe and search for the existence of the tables `concept`, `concept_ancestor` and `MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE`, under the `table_id` column. If they exists, you can look for the `row_count` field and check if it is `0`. Only then would you have to initiate the copy for the tables which are `0`

Reply:
This makes sense to me. I will work on this today.